docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-yp-issues.md

```diff
@@ -64,3 +64,23 @@ do
   ssh $IP firewall-cmd --zone=public --add-port=6379/tcp;
 done
 ```
+
+## Configure load balancer for Helm charts
+
+If you experience issues with configuring a load balancer when installing and configuring YugabyteDB Anywhere on Kubernetes, you should define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved.
```

Comment:
1. If you experience issues accessing YugabyteDB Anywhere through a Load Balancer, you should consider defining the Cross-Origin...
2. At the end, we could say
 
   For example, by adding a value like `--set additionalAllowedCorsOrigins:'https://mylbdomain'` to the appropriate helm command.